scarlet/cms/actions.py

```diff
@@ -75,11 +76,14 @@ def render(self, *args, **kwargs):
         return super(ActionView, self).render(*args, **kwargs)
 
     def get_navigation(self):
-        if not self.object and self.name not in self.bundle._meta.action_views:
-            if self.bundle.parent:
-                return self.bundle.parent.get_navigation(self.request, **self.kwargs)
-            return None
-        return super(ActionView, self).get_navigation()
+        nav = None
+        if self.show_navigation:
+            if not self.object and self.name not in self.bundle._meta.action_views:
+                if self.bundle.parent:
+                    nav = self.bundle.parent.get_navigation(self.request, **self.kwargs)
+            else:
+                nav = super(ActionView, self).get_navigation()
```

Comment:
while upgrade django and python, I would suggest to make py3 syntax and use short version
`nav = super().get_navigation()`